lib/sqewer/simple_job.rb

```diff
@@ -4,6 +4,13 @@
 # * initialize() will have keyword access to all accessors, and will ensure you have called each one of them
 # * to_h() will produce a symbolized Hash with all the properties defined using attr_accessor, and the job_class_name
 # * inspect() will provide a sensible default string representation for logging
+#
+# This module validates if the attributes defined in the job class are the same as
+# those persisted in the queue. More details on `Sqewer::SimpleJob#initialize`.
+# Because of this, it's required to create a new job class when adding of removing
+# an attribute.
+# This mechanism guarantees a strong consistency otherwise, a new deployed job class
+# could process old incompatible payloads.
```

Comment:
```suggestion
# This module validates if the attributes defined in the job class are the same as
# those persisted in the queue. More details on `Sqewer::SimpleJob#initialize`.
# Because of this, it's required to create a new job class when adding or removing
# an attribute.
# This mechanism guarantees strong consistency. Without it, a new deployed job class
# could process old incompatible payloads.
```

Reply:
thank you!!